pkg/adapter/adapter.go

```diff
@@ -355,6 +401,60 @@ func (a *Adapter) postMessage(msg wabbit.Delivery) error {
 	return nil
 }
 
+func isBinary(headers wabbit.Option) bool {
+	return headers["ce-specversion"] != nil || headers["Ce-Specversion"] != nil
```

Comment:
The source is not doing any event translation, and since the content-type could be empty, is more secure to pick the specversion that is mandatory for all CloudEvents. So just to know if the message is a cloudevent and the source is just translating/forwarding it, is better to look on the specversion.

Reply:
I'm wondering about the spelling choices. The [`http` module's `Header` type](https://pkg.go.dev/net/http#Header) provides `Get`, `Set` functions that handle the casing. (As well as `Add` and `Values`, because HTTP headers can be repeated.)

Reply:
But here there are Wabbit Headers (basically map[string]interface{}) headers, so they are not the same as http. Don't know if the http `Get` and `Set` would work here tho hmmm